float32[6] lin_vel
float32[6] ang_vel
